### PR TITLE
Download ServiceMonitor

### PR DESCRIPTION
**Pull request by shirhatti**
Addresses #79

### CODE REVIEW

File: windowsservercore-1709/Dockerfile [hunk 1]
```diff
@@ -1,9 +1,10 @@
...
-    Add-WindowsFeature Web-Server \
-    Invoke-WebRequest https://dotnetbinaries.blob.core.windows.net/servicemonitor/2.0.1.1/ServiceMonitor.exe -OutFile C:\ServiceMonitor.exe
+RUN powershell -Command `
+    Add-WindowsFeature Web-Server; `
+    Invoke-WebRequest -UseBasicParsing -Uri "https://dotnetbinaries.blob.core.windows.net/servicemonitor/2.0.1.1/ServiceMonitor.exe" -OutFile "C:\ServiceMonitor.exe"
 
 EXPOSE 80
 
-ENTRYPOINT ["C:\\ServiceMonitor.exe", "w3svc"]
+ENTRYPOINT ["C:\\ServiceMonitor.exe", "w3svc"]
```
Comment (shirhatti) [inline windowsservercore-1709/Dockerfile]: Nit: newline

File: windowsservercore-ltsc2016/Dockerfile [hunk 2]
```diff
@@ -1,9 +1,10 @@
...
-    Add-WindowsFeature Web-Server \
-    Invoke-WebRequest https://dotnetbinaries.blob.core.windows.net/servicemonitor/2.0.1.1/ServiceMonitor.exe -OutFile C:\ServiceMonitor.exe
+RUN powershell -Command `
+    Add-WindowsFeature Web-Server; `
+    Invoke-WebRequest -UseBasicParsing -Uri "https://dotnetbinaries.blob.core.windows.net/servicemonitor/2.0.1.1/ServiceMonitor.exe" -OutFile "C:\ServiceMonitor.exe"
 
 EXPOSE 80
 
-ENTRYPOINT ["C:\\ServiceMonitor.exe", "w3svc"]
+ENTRYPOINT ["C:\\ServiceMonitor.exe", "w3svc"]
```
Comment (shirhatti) [inline windowsservercore-ltsc2016/Dockerfile]: Nit: newline

File: windowsservercore-1709/Dockerfile (hunk 1 above)
Comment (IanOfOz) [reply windowsservercore-1709/Dockerfile]: Why is the windowsservercore-1709 Dockerfile now coming FROM the ltsc2016 base image? Isn't that going to double the image size?

Comment (shirhatti) [reply windowsservercore-1709/Dockerfile]: @IanOfOz Thanks for catching that. It was an accident that has been fixed in 619c3d052d3828e25f318e740fb051c5295f1e8c

**Pull request merged**